src/pages/AvailabilityPageMultiple.tsx

```diff
@@ -158,6 +158,7 @@ const AvailabilityPageMultiple = ({ event, refetchEvent }) => {
           `}
         >
           <SelectionTable
+            hover
```

Comment:
I don't think this is doing anything.
I didn't see `hover` prop being used in `SelectionTable` ?